app_dart/lib/src/datastore/cocoon_config.dart

```diff
@@ -160,6 +160,11 @@ class Config {
   String get flutterGoldChanges => 'Image changes have been found for '
       'this pull request.';
 
+  String get flutterGoldDraftChange => 'This pull request has been changed to a'
```

Comment:
space at the end? and maybe `'changed to draft status'` rather than `'changed to a draft status'`?

Reply:
or `'changed to a draft'`

Reply:
Done! Thanks!